tests/fhirrs/test_condition.py

```diff
@@ -1083,6 +1147,59 @@ def test_convert_condition_encounter_diagnosis2(
     assert diff == "", diff
 
 
+# similar to test_convert_condition_encounter_diagnosis3, except conditionDiagnosisUse has no
```

Comment:
Think you meant to say "similar to ...2"

Reply:
Good catch.  Fixed.